src/Stratis.Bitcoin.IntegrationTests/SegWitTests.cs

```diff
@@ -14,6 +14,8 @@ namespace Stratis.Bitcoin.IntegrationTests
 {
     public class SegWitTests
     {
+        // TODO: This test incorrectly assumes that SegWit is available on Stratis.
```

Comment:
I think you are misunderstanding the test, this test is to determine whether an SBFN node running on a Bitcoin network gets Segwit activated. Similarly below.

Reply:
Ah I see: `CoreNode stratisNode = builder.CreateStratisPowNode(KnownNetworks.RegTest);`

Ok.